### PR TITLE
MVC folder structure approach

### PR DESCRIPTION
**Pull request by Abiodun-Shittu**

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -19,60 +19,24 @@ const validateUser = (req, res, next) => {
...
-    const email = req.body.email;
-    const user = {
-        id: uuidv4(),
-        email,
-        name,
-    };
-    users.push(user);
-    res.status(200).json(user);
-});
+router.post('/users', validateUser, userController.createUser);
```
Comment (IyiKuyoro) [inline routes/users.js]: The same way you created the `controllers` folder and the `userController.js` file, I want you to create a `middlewares` folder and a `userMiddlewares.js` file and place this `validateUser` middleware in there.

**Review by IyiKuyoro — COMMENTED** [review]
Nice work on this. I have just one comment.

**Comment by IyiKuyoro** [comment]
Outside the PR review, I want you to add a PR description template that you will use to fill out your PR descriptions. Every PR must have a well-detailed description. Here is a link that can help you understand how to set up the PR description template. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

Create the template as a hidden file in the `.github` folder. You will understand this when you read document in that link. Here is the template I want you to use.

```markdown
## <PR Title> - <Jira ticket id linked>

### Changes To Be Made By This PR
- <list the changes made in this PR>

### PR Review Steps
- <provide a step by step guide on how to review this PR>
```

File: routes/users.js (hunk 1 above)
Comment (Abiodun-Shittu) [reply routes/users.js]: i was going to ask about where to place the validation, that got me thinking awhile

**Comment by courage173** [comment]
I think this looks good now!

**Review by IyiKuyoro — APPROVED** [review]

**Pull request merged**